.gitattributes

```diff
@@ -141,6 +141,8 @@ templates/lattice_susy_scale_constraint.hpp.in           export-ignore
 
 test                            export-ignore
 
+utils/loop_decays               export-ignore
```

Comment:
Why do you not export this directory? Can the user still create the decays without it?

Reply:
Of course. That directory contains utilities that pre-generate following files
- one_loop_decay_diagrams.cpp.in
- one_loop_decay_diagrams.hpp.in
- generic_loop_decay_diagram_classes.m

The user never has to use anything from it.